core/container/src/main/java/org/wildfly/swarm/container/runtime/cdi/CommonBean.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright 2015-2017 Red Hat, Inc, and individual contributors.
```

Comment:
Would this class and the Builder be better placed in the /spi?

Would enable fractions to utilize the bean creation pieces as well if they needed to